packages/components/src/components/primitives/IconButton/index.ts

```diff
@@ -0,0 +1 @@
+export * from './IconButton';
```

Comment:
`IconButtonProps` should be exported too

Reply:
fixed!